spec/controllers/v0/yellow_ribbon_programs_controller_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::YellowRibbonProgramsController, type: :controller do
+  context 'when determining version' do
+    it 'uses a production version as a default' do
+      create(:version, :production)
+      preview = create(:version, :preview)
+      create(:yellow_ribbon_program)
+      create(:yellow_ribbon_program, version: preview.number)
+      get(:index)
+      body = JSON.parse response.body
+      # we have one record associated with the production version, so we expect one result.
+      # this fails because we are getting back both the production and preview records
```

Comment:
Can you remove these comments?  They aren't accurate now since the spec does not fail (which is what we want).